environments/swiss/public.yml

```diff
@@ -59,11 +59,15 @@ formplayer_db_name: formplayer
 
 postgresql_dbs:
   - name: commcarehq_ucr
+    query_stats: True
+    host: "{{ groups.postgresql.0 }}"
```

Comment:
my guess is the role complained if hosts wansn't defined here? we  did the same in PNA so i guess this is the way to go.

Reply:
Currently hosts defaults to `localsettings.PG_DATABASE_HOST` though I'm if we get rid of that default (and probably the `localsettings.PG_DATABASE_HOST` altogether).